ios/CodePush/CodePushPackage.m

```diff
@@ -292,26 +292,37 @@ + (NSString *)getCodePushPath
 
 + (NSDictionary *)getCurrentPackage:(NSError **)error
 {
-    NSString *packageHash = [CodePushPackage getCurrentPackageHash:error];
-    if (*error || !packageHash) {
+    NSError *localError;
+    NSString *packageHash = [CodePushPackage getCurrentPackageHash:&localError];
```

Comment:
What would be the drawback if we simply passed `error` instead of `&localError` here, if we know that `getCurrentPackageHash` already guards against nil dereferencing `error`?